lib/application/usecases/log/GetLogUseCase.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+const { IUseCase } = require('../../interfaces');
+const {
+    repositories: {
+        LogRepository,
+    },
+    utilities: {
+        TransactionHelper,
+    },
+} = require('../../../database');
+
+/**
+ * GetLogsUseCase
+ */
+class GetLogsUseCase extends IUseCase {
```

Comment:
Im not sure if the plural form fits here

Reply:
No, it doesn't 😄 Good catch